### ISSUE TITLE
Infer trained model on OpenVino

### ISSUE THREAD

**Issue by sudapure** [lars76/object-localization#17]
Is it possible to infer trained model in openvino or even in opencv dnn module, as the openvino supports standard object detection models like, mobilenet with SSD, yolo, mask-rccn,etc. and simple classification models with respective neural network architecture(ex. inception,mobilenet,etc), so is it possible to convert h5 keras models to convert into native tensorflow frozen graph and infer it on openvino.
